app/controllers/content_items_controller.rb

```diff
@@ -61,6 +62,16 @@ def sign_in_options
     end
   end
 
+  def self_assessment_start_page?(content_item)
+    self_assessment_pages = %w(
+      /log-in-file-self-assessment-tax-return
+      /log-in-file-self-assessment-tax-return/choose_sign_in
```

Comment:
Should this be `/log-in-file-self-assessment-tax-return/choose-sign-in`?

Reply:
Yes! Thank you. Will fix.